src/docfx/build/page/BuildPage.cs

```diff
@@ -234,15 +234,14 @@ private static (List<Error> errors, JObject model, InputMetadata inputMetadata)
                 var landingData = (LandingData)content;
                 JsonUtility.Merge(inputMetadata.RawJObject, landingData.ExtensionData);
 
-                pageModel = new JObject()
+                pageModel = JsonUtility.ToJObject(new ConceptualModel
                 {
-                    ["conceptual"] = HtmlUtility.LoadHtml(await RazorTemplate.Render(file.Mime, content)).HtmlPostProcess(file.Docset.Culture),
-                };
+                    Conceptual = HtmlUtility.LoadHtml(await RazorTemplate.Render(file.Mime, content)).HtmlPostProcess(file.Docset.Culture),
+                    Title = inputMetadata.Title ?? obj?.Value<string>("title"),
+                    RawTitle = $"<h1>{obj?.Value<string>("title")}</h1>",
+                });
             }
 
-            pageModel["title"] = inputMetadata.Title ?? obj?.Value<string>("title");
-            pageModel["rawTitle"] = file.Docset.Legacy ? $"<h1>{obj?.Value<string>("title")}</h1>" : null;
-
             return (errors, pageModel as JObject, inputMetadata);
```

Comment:
```suggestion
            return (errors, pageModel, inputMetadata);
```

Reply:
page model is not a JObject always, it can be a transformed token

Reply:
Need to check and report like I did [here](https://github.com/dotnet/docfx/pull/4810/files#diff-54e86e4d213ae7f283c6a9bf56b3f8f8R48) and [here](https://github.com/dotnet/docfx/pull/4810/files#diff-c329d0f4c7d5076b957537c2818b4a03R238)